CHANGELOG.md

```diff
@@ -10,9 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 ### Added
+- Add in-enclave segfault backtrace log messages when the enclave is configured with Debug=1 and CapturePFGPExceptions=1, and the in-enclave logging is enabled with level=INFO.
 
 ## Changed
 - Fix the incorrect behavior of pthread_mutex_init() and std::mutex such that they no longer create a recursive lock by default.
+- Fix the bug in oesign that always sets CapturePFGPExceptions=1 when the configuration presents in the config file regardless the value specified.
+- Fix the behavior of hanging when vDSO is used and the segfault occurs inside the enclave. Now the application will terminate and return OE_FAILURE.
```

Comment:
Is the fix applicable only for debug mode enclaves? The fixed behavior seems appropriate for Producetion enclaves as well.

Reply:
This is applicable for both debug and production enclaves.